File: FDP/T3_A5_PinedoBerumenClaraRuby/src/Ejercicio02_Divisores.java

import java.util.Scanner;

public class Ejercicio02_Divisores {

	public static void main(String[] args) {
		
		Scanner entrada = new Scanner(System.in);
		
		System.out.println("==Programa que muestra los divisores de un número==");
		System.out.println();
		
		
		System.out.println("Ingresa el número: ");
		int num = entrada.nextInt();
		
		System.out.println("Los divisores de " + num + " son: ");
		
		for(int i=1; i<=num/2; i++ ) {
			if(num % i== 0) 
				System.out.println(i);
			
		}
		
		System.out.println(num);
	
		


	}

}

File: FDP/T5_A5_PinedoBerumenClaraRuby/Ejercicio02_Divisores.py
'''
Created on 4 dic. 2019

@author: Ruby
'''
print("Programa que muestra los divisores de un numero ingresado desde teclado\n")

numero = int (input("Ingresa un numero: "))

print(f"Los divisores del numero son: ")

for i in range(1,numero):
    if (numero % i == 0):
        print(int(i))
print(numero)



